dev/breeze/tests/test_selective_checks.py

```diff
@@ -371,7 +371,7 @@ def assert_outputs_are_printed(expected_outputs: dict[str, str], stderr: str):
         ),
         (
             pytest.param(
-                ("providers/tests/system/apache/beam/file.py",),
+                ("providers/apache/beam/tests/file.py",),
```

Comment:
@potiuk @eladkal Shall I remove this `pytest.param` config altogether? I am not able to figure out why only `apache.beam common.compat google` are in the `selected-providers-list-as-string`.